sdk/messaging/azservicebus/stress/stress.go

```diff
@@ -24,9 +26,15 @@ import (
 // | summarize Sum=sum(valueCount) by bin(timestamp, 30s), name
 // | render timechart
 
-var messagesSent int32
-var messagesReceived int32
-var exceptions int32
+type Stats struct {
```

Comment:
This is all currently part of public surface area, is that intentional?

Reply:
No, unexported